bad.py

```diff
@@ -0,0 +1,4 @@
+5 == 5
+
+concat_list = ["str1" "str2", "str3"]
+6 == 6
```

Comment:
## semgrep-action
**Severity:** Error :red_circle:
**Rule id:** python.lang.correctness.useless-eqeq.useless-eqeq
**Message:** This is always True: `6 == 6` or `6 != 6`. If testing for floating point NaN, use `math.isnan(6)`,
or `cmath.isnan(6)` if the number is complex.

<sub>[Report an issue with this rule.](https://github.com/returntocorp/semgrep-rules/issues/new?assignees=&labels=&template=bug_report.md&title=python.lang.correctness.useless-eqeq.useless-eqeq)</sub>